pkg/node/manager/manager_test.go

```diff
@@ -73,20 +74,43 @@ func newIPcacheMock() *ipcacheMock {
 	}
 }
 
+func AddrOrPrefixToIP(ip string) (net.IP, error) {
```

Comment:
Could this be useful to move to netip helpers?

Reply:
I looked back over this, and this is not using the `netip` structures. Since it's based on the older type and only used in this testing package until we migrate this testing code over to `netip`, I think it is not worth migrating to a more central location.